internal/handlers/task_handler.go

```diff
@@ -12,65 +12,55 @@ import (
 	"time"
```

Comment:
add an new package called "services" and divide the logic in the handler to:
1- Request validation and response handling in the handler
2- All the other logic move it to the service